### ISSUE TITLE
Crash with JEI dragging

### ISSUE THREAD

**Issue by Darkere** [desht/ModularRouters#91]
### Mod Version 1.16.1-7.0.1-29

Dragging fluids from JEI (instead of the bucket item).

```
Caused by: java.lang.ClassCastException: net.minecraftforge.fluids.FluidStack cannot be cast to net.minecraft.item.ItemStack
	at me.desht.modularrouters.integration.jei.GuiModuleGhost$ItemTarget.accept(GuiModuleGhost.java:32) ~[?:1.16.1-7.0.1-29] {re:classloading}
	at mezz.jei.gui.ghost.GhostIngredientDrag.onClick(GhostIngredientDrag.java:103) ~[?:7.0.1.10] {re:classloading}
	at mezz.jei.gui.ghost.GhostIngredientDragManager.handleMouseClicked(GhostIngredientDragManager.java:82) ~[?:7.0.1.10] {re:classloading}
	at mezz.jei.gui.overlay.IngredientListOverlay.handleMouseClicked(IngredientListOverlay.java:246) ~[?:7.0.1.10] {re:classloading}
	at mezz.jei.input.InputHandler.handleMouseClick(InputHandler.java:156) ~[?:7.0.1.10] {re:classloading}
	at mezz.jei.input.InputHandler.onGuiMouseEvent(InputHandler.java:121) ~[?:7.0.1.10] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_1306_InputHandler_onGuiMouseEvent_Pre.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-3.0.3-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:297) ~[eventbus-3.0.3-service.jar:?] {}
	at net.minecraftforge.client.ForgeHooksClient.onGuiMouseClickedPre(ForgeHooksClient.java:637) ~[?:?] {re:classloading}
	at net.minecraft.client.MouseHelper.lambda$mouseButtonCallback$0(MouseHelper.java:86) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.screen.Screen.func_231153_a_(Screen.java:421) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
```

https://gist.github.com/Darkere/8ebe103bbdc1d1e2867b28221afd5e2f

**Comment by desht** [comment]
Build 34 from https://jenkins.k-4u.nl/job/Modular%20Routers%201.14+/ has this fixed.

**Comment by desht** [comment]
Fixed in 7.1.1 release
